Status: Succeeded
Test Event Name: Test

Response:
{
  "status": "IN_PROGRESS",
  "message": "Schedule file 'SCH_HHUN_20250515_Final_v1.xml' uploaded but processed file not confirmed within timeout.",
  "archived_json": "s3://dev-playout-schedule-943412361827/tx-schedules/archive/SCH_HHUN_20250515_Final_v1.json",
  "vipe_json": "s3://uat-inbound-media-files-980921750886/rt-demo/schedules/ARQTV3/SCH_HHUN_20250515_Final_v1.json",
  "file_name": "SCH_HHUN_20250515_Final_v1.xml",
  "channel": "ARQTV3",
  "polling_enabled": true,
  "processed_found": false,
  "processed_path": "procschedules/ARQTV3/done/"
}

Function Logs:
o identity-based policy allows the s3:ListBucket action
2025-08-14 21:02:25.942 | WARNING  | app:poll_for_processed_schedule:131 - Error checking for files in path rt-demo/procschedules/ARQTV3/done/: An error occurred (AccessDenied) when calling the ListObjectsV2 operation: User: arn:aws:sts::980921750886:assumed-role/dev-vipe-cross-account-role/VipeS3UploadSession is not authorized to perform: s3:ListBucket on resource: "arn:aws:s3:::uat-inbound-media-files-980921750886" because no identity-based policy allows the s3:ListBucket action
2025-08-14 21:02:30.948 | WARNING  | app:poll_for_processed_schedule:131 - Error checking for files in path rt-demo/procschedules/ARQTV3/done/: An error occurred (AccessDenied) when calling the ListObjectsV2 operation: User: arn:aws:sts::980921750886:assumed-role/dev-vipe-cross-account-role/VipeS3UploadSession is not authorized to perform: s3:ListBucket on resource: "arn:aws:s3:::uat-inbound-media-files-980921750886" because no identity-based policy allows the s3:ListBucket action
2025-08-14 21:02:35.965 | WARNING  | app:poll_for_processed_schedule:131 - Error checking for files in path rt-demo/procschedules/ARQTV3/done/: An error occurred (AccessDenied) when calling the ListObjectsV2 operation: User: arn:aws:sts::980921750886:assumed-role/dev-vipe-cross-account-role/VipeS3UploadSession is not authorized to perform: s3:ListBucket on resource: "arn:aws:s3:::uat-inbound-media-files-980921750886" because no identity-based policy allows the s3:ListBucket action
2025-08-14 21:02:40.972 | WARNING  | app:poll_for_processed_schedule:131 - Error checking for files in path rt-demo/procschedules/ARQTV3/done/: An error occurred (AccessDenied) when calling the ListObjectsV2 operation: User: arn:aws:sts::980921750886:assumed-role/dev-vipe-cross-account-role/VipeS3UploadSession is not authorized to perform: s3:ListBucket on resource: "arn:aws:s3:::uat-inbound-media-files-980921750886" because no identity-based policy allows the s3:ListBucket action
2025-08-14 21:02:45.972 | WARNING  | app:poll_for_processed_schedule:134 - Timeout reached. File not found after 52.527 seconds.
2025-08-14 21:02:45.972 | WARNING  | app:handler:306 - Processed schedule not found for SCH_HHUN_20250515_Final_v1.xml in channel ARQTV3 within timeout period
2025-08-14 21:02:45.972 | INFO     | core.database:get_database_credentials:27 - Fetching new database credentials from Secrets Manager: arn:aws:secretsmanager:eu-west-1:943412361827:secret:/database/dev/base/master_credentials-5gtulW
2025-08-14 21:02:46.314 | INFO     | core.database:get_database_connection:57 - Creating new connection pool for database 'playout_dev_db_provisioned' via proxy 'dev-base-playout-aurora.cluster-clqeoqmuuyhc.eu-west-1.rds.amazonaws.com'...
2025-08-14 21:02:46.315 | INFO     | core.database:get_database_connection:60 - Database connection pool created successfully.
2025-08-14 21:02:46.523 | INFO     | repositories.work_order:get_by_id:106 - Executing GET query for work_order with ID: 3259eeb5-62dc-416a-8e05-51b6be20fa6d
2025-08-14 21:02:46.618 | INFO     | repositories.work_order:update_status:136 - Updating status of work order 3259eeb5-62dc-416a-8e05-51b6be20fa6d to IN_PROGRESS
2025-08-14 21:02:46.646 | INFO     | repositories.work_order:update_status:144 - Successfully updated work order 3259eeb5-62dc-416a-8e05-51b6be20fa6d status to IN_PROGRESS
2025-08-14 21:02:46.647 | INFO     | work_order_status_updater:update_work_order_status:59 - Successfully updated status of work order 3259eeb5-62dc-416a-8e05-51b6be20fa6d to IN_PROGRESS
2025-08-14 21:02:46.647 | INFO     | work_order_status_updater:update_work_order_status:68 - Database connection closed successfully.
2025-08-14 21:02:46.648 | ERROR    | app:handler:315 - Failed to update work order status for 3259eeb5-62dc-416a-8e05-51b6be20fa6d: a coroutine was expected, got None
END RequestId: 8c816839-ccd0-4a02-b520-35a5ed8a6606
REPORT RequestId: 8c816839-ccd0-4a02-b520-35a5ed8a6606	Duration: 58262.62 ms	Billed Duration: 58263 ms	Memory Size: 512 MB	Max Memory Used: 163 MB	Init Duration: 1318.85 ms

Request ID: 8c816839-ccd0-4a02-b520-35a5ed8a6606
